gems/pending/disk/modules/AzureBlobDisk.rb

```diff
@@ -72,7 +72,7 @@ def blob_read(start_byte, length)
     ret = @storage_acct.get_blob_raw(@container, @blob, key, options)
 
     content_md5  = ret.headers[:content_md5].unpack("m0").first.unpack("H*").first
-    returned_md5 = Digest::MD5.hexdigest(ret.body)
+    returned_md5 = Rails.application.config.digest_class.hexdigest(ret.body)
```

Comment:
This is one that's not changeable, for example.  We receive the md5 from azure, and then compare it to our calculated digest.  The only way to avoid md5 here is to not do it at all, which may not be acceptable.